src/components/Card.js

```diff
@@ -2,20 +2,38 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import emoji from 'emoji-dictionary';
 
+
 import './Card.css';
 
+function convertToemoji(anEmoji){
+  if (anEmoji != null){
+   return emoji.getUnicode(anEmoji)
+  }
+}
+
+
 class Card extends Component {
+   findID = () => {
```

Comment:
not a very great function name, maybe something like just `deleteCard` would be better